docs/tools/debugging/pro.md

```diff
@@ -0,0 +1,106 @@
+---
+order: 2
+---
+
+# Debug Like A Pro
+
+## Intro
+
+Tendermint Core is a fairly robust BFT replication engine. Unfortunately, as with other software, failures sometimes do happen. The question is then “what do you do” when the system deviates from the expected behavior.
+
+The first response is usually to take a look at the logs. By default, Tendermint writes logs to standard output ¹.
```

Comment:
Since the footnote here isn't a link, it's probably better to just replace it with something like:

```suggestion
The first response is usually to take a look at the logs. By default, Tendermint writes logs to standard output, but alternatively you can redirect the output to a file or forward it to another server.
```